windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -22,8 +22,15 @@
     msu_file_path: ""
     msu_nfs_path_list: []
 
+# For OS older than 17763 (Windows 1809 or Windows Server 2019), it has very high frequency of timeout for the command.
 - name: "Get unused driver letter"
   include_tasks: ../utils/win_get_unused_drive_letter.yml
+  when: guest_os_build_num | int >= 17763
+
+- name: "Set the new driver letter for older OS"
```

Comment:
The `older OS` in the task name can be replaced with guest OS version or build

Reply:
Updated. Thanks.